.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[target.aarch64-apple-darwin]
+rustflags = [
+  "-C", "link-arg=-L/opt/homebrew/opt/libomp/lib"
```

Comment:
build.rs also modifies the search path, and the following segment is already surrounded by special cases for macos, linux and windows

https://github.com/DeepSignSecurity/lightgbm-rs/blob/820a4604e842a749134b6377e1253b47e84fbafd/lightgbm-sys/build.rs#L67-L68

Would it work if we add it there, along with a comment why it's useful?

Reply:
Ah yeah, we could probably do that too. 
I ended up creating a config.toml because that's what I'm used to but yeah, build.rs probably makes more sense.